src/app/components/address-displayer/form-address-displayer.tsx

```diff
@@ -15,9 +15,8 @@ export function FormAddressDisplayer({ address, ...rest }: FormAddressDisplayerP
       justifyContent="end"
       maxWidth="300px"
       mr="-8px"
-      {...rest}
     >
-      <AddressDisplayer address={address} />
+      {address && <AddressDisplayer address={address} />}
```

Comment:
It's more guesswork/ clutching at straws so I can roll it back? 

I added this as sometimes we map and [pass](https://github.com/leather-wallet/extension/blob/9f73b24e6f16970f1d7355956dd918bec8bb5f85/src/app/pages/rpc-send-transfer/rpc-send-transfer-summary.tsx#L73 ) `recipient.address`. 

I was searching for the cause of a possible `"Cannot read property of undefined reading .address" error.` It should throw a type error but I'm not sure if that will help if we are mapping through recipients with an undefined address.

Reply:
Would prefer to revert this change. This would really confuse me reading this code. 

If investigating, suggesting put a analytics event here if undefined? Or, to do a runtime check on this, do it where the component is used vs inside it.